airbyte-integrations/connector-templates/source-python/Dockerfile

```diff
@@ -1,13 +1,27 @@
-FROM python:3.7-slim
+FROM python:3.7.11-alpine3.14 as base
 
-# Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+# build and load all requirements 
+FROM base as builder
+WORKDIR /airbyte/integration_code
+
+# upgrade pip to the latest version
+RUN apk --no-cache upgrade \
+    && pip install --upgrade pip
 
+COPY setup.py ./
```

Comment:
Should we copy main.py as well?

Reply:
for installation of all requirements we need a setup.py file only.
Sure we can add payload files too but they will not be used throughout this building stage.